lib/vdc/ArbitrarilyOrientedRegularGrid.cpp

```diff
@@ -0,0 +1,228 @@
+#include <limits>
+#include <iomanip>
+
+#include <vapor/utils.h>
+
+#include <glm/gtc/type_ptr.hpp>
+#include <GTE/MinimumAreaBox2.h>
+#include <vapor/RegularGrid.h>
+#include <vapor/ArbitrarilyOrientedRegularGrid.h>
+
+using namespace std;
+using namespace VAPoR;
+
+// clang-format off
+
+ArbitrarilyOrientedRegularGrid::ArbitrarilyOrientedRegularGrid(
+    const VAPoR::Grid *grid3d,
+    planeDescription& pd,
+    const DimsType& dims,
+    std::shared_ptr<float>& blks
```

Comment:
Isn't this argument now obsolete? That was the entire point of having ArbitrarilyOrientedRegularGrid manage the space.

Reply:
Yep.  This change was on a separate branch but not this one.  Updated and thanks for the catch.